resources/RDcode.py

```diff
@@ -219,6 +222,59 @@ def create_Customer(RECORD_COUNT,Lc):
     df = pd.DataFrame(data_value, columns = columns)
     return df
 
+def create_Suppiler(RECORD_COUNT,Lc):
+    fake = Faker()
+    df=pd.DataFrame(columns=['Supplier_ID','SupplierName','Location','Address','States','Lat','Long'])
+    data=Lc
+    for i in range(RECORD_COUNT):
+        indexlocation=np.random.choice(len(data))
+        Supplier_ID=i
+        SupplierName=fake.company()
+        Location_ID=data['Location_ID'][indexlocation]
+        Address=data['Address'][indexlocation]
+        States=data['States'][indexlocation]
+        Lat=data['Lat'][indexlocation]
+        Long=data['Long'][indexlocation]
+        to_append=[Supplier_ID, SupplierName, Location_ID, Address,States,Lat, Long]
+        a_series = pd.Series(to_append, index = df.columns)
+        df = df.append(a_series, ignore_index=True) 
+    return(df)
+
+
+def create_Storage(RECORD_COUNT,Pr,Wa):
+    fake = Faker()
+    df=pd.DataFrame(columns=['Storage_ID','Product','Warehouse','Capability','Quantity'])
+    for i in range(RECORD_COUNT):
+        Storage_ID=i
+        Product=random.randint(1, len(Pr)-1)
+        Warehouse=random.randint(1, len(Wa)-1)
```

Comment:
Warehouse_ID starts at 0, so random should starts at 0.
random.randint(0, len(Wa)-1)